base_transaction_id/models/invoice.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+
+from odoo import fields, models
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.move"
+
+    transaction_id = fields.Char(
```

Comment:
The suffix _id is kind of confusing here (@gurneyalex :thinking:), not an issue related to the module migration however